griptape/memory/structure/base_conversation_memory.py

```diff
@@ -88,7 +88,7 @@ def add_to_prompt_stack(self, prompt_stack: PromptStack, index: Optional[int] =
 
         if num_runs_to_fit_in_prompt:
             memory_inputs = self.to_prompt_stack(num_runs_to_fit_in_prompt).messages
-            if index:
+            if index is not None:
```

Comment:
This is why we check is not None!

Reply:
Can you flip the if-else to be `if is None ... else ...` for better readability?

Reply:
One of these days I'll get it first try...

I wonder if there's a linter rule for this?